tests/integration/project_test.py

```diff
@@ -439,3 +445,85 @@ def test_unscale_after_restart(self):
         project.up()
         service = project.get_service('web')
         self.assertEqual(len(service.containers()), 1)
+
+    def test_project_up_volumes(self):
+        vol_name = 'composetests_{0:x}'.format(random.getrandbits(32))
```

Comment:
Bit strange that this comes out as `composetest_composetests_<hexchars>` - I don't think two prefixes are necessary